controllers/transactions.js

```diff
@@ -0,0 +1,33 @@
+const transactionsModel = require('../models/transaction')
+const { getUserId } = require('../utils/users')
+
+/**
+ * Collects all transactions and sends only required data for username spicified in url
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const fetch = async (req, res) => {
+  try {
+    const userId = await getUserId(req.params.username)
+    if (userId) {
+      const orderBy = parseInt(req.query.orderBy, 10) || 'DESC'
```

Comment:
How is `orderBy` going to be parsed into integer? 🤔

Reply:
Oh ...my bad

Reply:
parseInt removed